src/brogue/RogueMain.c

```diff
@@ -1217,13 +1223,19 @@ void victory(boolean superVictory) {
             printString(buf, mapToWindowX(60), min(ROWS-1, i + 1), &itemMessageColor, &black, dbuf);
             totalValue += max(0, itemValue(theItem) * 2);
             i++;
-        } else if (itemValue(theItem) > 0) {
+        } else {
             identify(theItem);
-            itemName(theItem, buf, true, true, &white);
+            itemName(theItem, buf, false, true, &white);
             upperCase(buf);
-            printString(buf, mapToWindowX(2), min(ROWS-1, i + 1), &white, &black, dbuf);
-            sprintf(buf, "%li", max(0, itemValue(theItem)));
-            printString(buf, mapToWindowX(60), min(ROWS-1, i + 1), &itemMessageColor, &black, dbuf);
+
+            sprintf(buf2, "%c - %s", theItem->displayChar, buf);
```

Comment:
`%c` is not appropriate for displaying the data type of displayChar, which is technically `unsigned short`.

If you're going to list items, you should do so consistently with the inventory interface, i.e. write the icons in yellow, and not use `-` to separate them from text, etc.

Reply:
Ok, I will reproduce the same inventory list, logic.